### ISSUE TITLE
2nd click on menubar icon, causes it to disappear from menubar

### ISSUE THREAD

**Issue by mvgfr** [musa11971/Music-Bar#57]
Relaunching the app results in a repeat of the behavior; I was only able to get it to work on the first click.
ENV: macOS 10.15.7 (19H15) on MacBookPro15,1

**Comment by porcherie** [comment]
same problem here on same OS but on MacBook Pro late 2012. Any solution?

**Comment by greycubesgav** [comment]
Similar issue here, I run MusicBar, click the icon, the app crashes.
Re-running shows the same issue.

Here's a quick screen grab of the issue visually.
https://user-images.githubusercontent.com/3426859/105208281-32efff80-5b51-11eb-998f-cb257c943575.mov

mac Version: macOS Catalina 10.15.6
MusicBar Version: 1.4

**Comment by greycubesgav** [comment]
Attaching macos .crash file:
[Music Bar_2021-01-20-185517_Gavins-iMac.crash.txt](https://github.com/musa11971/Music-Bar/files/5843975/Music.Bar_2021-01-20-185517_Gavins-iMac.crash.txt)

**Comment by dodger** [comment]
I think I have the same thing here. A click on the icon when music isn't playing crashes it, and a click on the square when music _is_ playing crashes it. I can, however, click on the song title when it's playing and open the square, and I can scrub.

**Comment by meyerweb** [comment]
> Similar issue here, I run MusicBar, click the icon, the app crashes.
> Re-running shows the same issue.
> 
> Here's a quick screen grab of the issue visually.
> https://user-images.githubusercontent.com/3426859/105208281-32efff80-5b51-11eb-998f-cb257c943575.mov
> 
> mac Version: macOS Catalina 10.15.6
> MusicBar Version: 1.4

I get the exact same experience, except I’m using Mojave 10.14.6 (18G8022) on a MacBook Pro (Retina, 15-inch, Early 2013).  Happy to attach a crash report if needed, and someone reminds me where to find it.

**Comment by jakekiley** [comment]
Same issue, and specs as mvgfr 
> Relaunching the app results in a repeat of the behavior; I was only able to get it to work on the first click.
ENV: macOS 10.15.7 (19H15) on MacBookPro15,1

Commenting to share that reverting to the previous release works fine in case that helps troubleshoot or allows others to use the app. 
https://github.com/musa11971/Music-Bar/releases/tag/v1.3.3

**Comment by musa11971** [comment]
@kiritokatklian and I took a look at this, but cannot seem to replicate the issue, nor do we know what is causing this. We would love to apply a fix, but we need more information to identify the issue. 

Any crash reports or additional information will help.

**Comment by svenrat** [comment]
Same issue here (10.15.7) with v1.4 while v1.3.3 works fine. Then I've started v1.4 with "`/Applications/Music\ Bar.app/Contents/MacOS/Music\ Bar`" in a Terminal. It starts and when I click on the icon in the menu the program crashes and in the terminal I see the following error:

```
2021-07-13 18:57:45.254 Music Bar[1130:62043] Could not find image named 'Symbols/pause.fill'.
2021-07-13 18:57:45.255 Music Bar[1130:62043] Could not find image named 'Symbols/backward.fill'.
2021-07-13 18:57:45.255 Music Bar[1130:62043] Could not find image named 'Symbols/forward.fill'.
2021-07-13 18:57:45.258 Music Bar[1130:62043] Could not find image named 'Symbols/gear'.
zsh: illegal hardware instruction  /Applications/Music\ Bar.app/Contents/MacOS/Music\ Bar
```

I think, that's the reason why it crashes when you start MusicBar BEFORE you start the MusicPlayer and why I can't see the navigation icons when you start MusicBar AFTER you start the MusicPlayer.
Hope this can help.

**Comment by musa11971** [comment]
Ok. Unfortunately still not able to replicate this on my end. However, assuming this is caused by the change in icon-types in v1.4, I have built a test version that reverts this change.

Could those who are having this issue please test [this build](https://github.com/musa11971/Music-Bar/files/6815276/Music.Bar-DEBUG.zip)? If this build fixes the issue, I will include the change in the next release.

Thanks.

**Comment by svenrat** [comment]
> Could those who are having this issue please test [this build](https://github.com/musa11971/Music-Bar/files/6815276/Music.Bar-DEBUG.zip)? If this build fixes the issue, I will include the change in the next release.

In my virtual enviroment it works now without any problems. Also when starting in a terminal there are no more error messages. I'll test it tonight on my "real" Mac when I'm at home.

**Comment by svenrat** [comment]
> > Could those who are having this issue please test [this build](https://github.com/musa11971/Music-Bar/files/6815276/Music.Bar-DEBUG.zip)? If this build fixes the issue, I will include the change in the next release.
> 
> In my virtual enviroment it works now without any problems. Also when starting in a terminal there are no more error messages. I'll test it tonight on my "real" Mac when I'm at home.

Testing the debug version on my Mac I can confirm that the problem is gone.

**Comment by musa11971** [comment]
Fix released in v1.5

**Comment by mvgfr** [comment]
late to the party, however I've now confirmed too - thx!
